packages/react-components/react-alert/Spec.md

```diff
@@ -1,41 +1,92 @@
-# @fluentui/react-alert Spec
+# Alert
 
 ## Background
 
-_Description and use cases of this component_
+An Alert displays a brief, important message to attract a user's attention without interrupting their current task.
 
 ## Prior Art
 
-_Include background research done for this component_
-
-- _Link to Open UI research_
-- _Link to comparison of v7 and v0_
-- _Link to GitHub epic issue for the converged component_
+- [Alert - Open UI](https://open-ui.org/components/toast.research)
+- [Convergence epic issue #22579](https://github.com/microsoft/fluentui/issues/22579)
 
 ## Sample Code
 
-_Provide some representative example code that uses the proposed API for the component_
+Default Alert
+
+```
+import { Alert } from '@fluentui/react-alert'
+
+const AlertExample = () => <Alert content="This is a default alert" />
+```
+
+Success Alert
+
+```
+import { Alert } from '@fluentui/react-alert'
+
+const AlertExample = () => <Alert intent="success" content="This is a success alert" />
```

Comment:
The whole spec describes `content` slot but implementation uses `children`. The spec should be updated.